src/main/java/com/volasoftware/tinder/controller/UserController.java

```diff
@@ -19,11 +19,19 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping("/api/v1/users/profile")
+    @GetMapping("/api/v1/users/show-profile")
     @SecurityRequirement(name = "Bearer Authentication")
     @ResponseBody
     public UserProfileDto getUserProfile() {
 
         return userService.getCurrentUserProfile();
     }
+
+    @PostMapping("/api/v1/users/edit-profile")
+    @SecurityRequirement(name = "Bearer Authentication")
+    public ResponseEntity editUser(UserProfileDto userProfileDto){
+        userService.editUserProfile(userProfileDto);
+
+        return ResponseEntity.ok(HttpStatus.OK );
```

Comment:
In many cases, we want to show updated object after the request. Please, return in the response updated user (wrapped in dto, of course)!